lib/timeline/timeline.dart

```diff
@@ -46,9 +46,29 @@ class TimelineScreen extends Screen {
   PButton resumeButton;
 
   @override
-  void createContent(Framework framework, CoreElement mainDiv) {
+  void createContent(Framework framework, CoreElement mainDiv) async {
     FrameDetailsUI frameDetailsUI;
 
+    final Isolate isolate = await serviceInfo.service
+        .getIsolate(serviceInfo.isolateManager.selectedIsolate.id);
+
+    bool isProfileMode;
```

Comment:
We should create a FlutterApp object that tracks whether the app is debug mode or not.